homeassistant/components/sensor/recollect_waste.py

```diff
@@ -0,0 +1,99 @@
+"""
+Support for Recollect Waste curbside collection pickup.
+
+For more details about this platform, please refer to the documentation at
+https://www.home-assistant.io/components/sensor.recollect_waste/
+"""
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
```

Comment:
Small nit, but let’s add a space below to separate third-party modules from those inside HASS itself.